src/Application.js

```diff
@@ -529,9 +531,14 @@ define('Mobile/SalesLogix/Application', [
             this.requestUserOptions();
             this.requestSystemOptions();
             this.setDefaultMetricPreferences();
+            this.initQuickFormServices();
         },
         onRequestUserDetailsFailure: function(response, o) {
         },
+        initQuickFormServices: function () {
+            var qfs = App.serviceManager.get('quickFormService');
+            qfs.initModelData();
```

Comment:
We should probably check this variable for null/undefined before calling methods on it